charts/neuvector/100.0.1+up2.2.2/questions.yaml

```diff
@@ -0,0 +1,341 @@
+- variable: enforcer.image.tag
+  default: ""
+  description: image tag for enforcer
+  type: string
+  label: Enforcer Image Tag
+  group: "Container Images"
+- variable: cve.scanner.image.repository
+  default: "neuvector/scanner"
+  description: scanner image repository
+  type: string
+  label: Scanner image path
+  group: "Container Images"
+- variable: cve.scanner.image.tag
+  default: ""
+  description: image tag for scanner
+  type: string
+  label: Scanner Image Tag
+  group: "Container Images"
+- variable: cve.updater.image.repository
+  default: "neuvector/updater"
+  description: cve updater image repository
+  type: string
+  label: CVE Updater image path
+  group: "Container Images"
+- variable: cve.updater.image.tag
+  default: ""
+  description: image tag for updater
+  type: string
+  label: Updater Image Tag
+  group: "Container Images"
+#Container Runtime configurations
+- variable: docker.enabled
+  default: true
+  description:  Docker runtime. Enable only one runtime.
+  type: boolean
+  label: Docker Runtime
+  show_subquestion_if: true
+  group: "Container Runtime"
+  subquestions:
+  - variable: docker.path
+    default: "/var/run/docker.sock"
+    description: "Docker Runtime Path"
+    type: string
+    label: Runtime Path
+- variable: containerd.enabled
+  default: "false"
+  description: Containerd runtime. Enable only one runtime.
+  type: boolean
+  label: Containerd Runtime
+  show_subquestion_if: true
+  group: "Container Runtime"
+  subquestions:
+  - variable: containerd.path
+    default: " /var/run/containerd/containerd.sock"
+    description: "Containerd Runtime Path"
+    type: string
+    label: Runtime Path
+- variable: crio.enabled
+  default: "false"
+  description: CRI-O runtime. Enable only one runtime.
+  type: boolean
+  label: CRI-O Runtime
+  show_subquestion_if: true
+  group: "Container Runtime"
+  subquestions:
+  - variable: crio.path
+    default: "/var/run/crio/crio.sock"
+    description: "CRI-O Runtime Path"
+    type: string
+    label: Runtime Path
+- variable: k3s.enabled
+  default: "false"
+  description: k3s containerd runtime. Enable only one runtime.
+  type: boolean
+  label: k3s Containerd Runtime
+  show_subquestion_if: true
+  group: "Container Runtime"
+  subquestions:
+  - variable: k3s.runtimePath
+    default: " /run/k3s/containerd/containerd.sock"
+    description: "k3s Containerd Runtime Path"
+    type: string
+    label: Runtime Path
+#storage configurations
+- variable: controller.pvc.enabled
+  default: false
+  description: If true, enable persistence for controller using PVC
+  type: boolean
+  label: PVC status
+  group: "PVC Configuration"
+- variable: controller.pvc.storageClass
+  default: ""
+  description: Storage Class to be used
+  type: string
+  label: Storage Class Name
+  group: "PVC Configuration"
+#ingress configurations
+- variable: manager.ingress.enabled
+  default: false
+  description: If true, create ingress, must also set ingress host value
```

Comment:
nit:
This file has inconsistent use of periods in setting descriptions.
As well as inconsistent capitalization of Ingress between labels and descriptions.

Reply:
I will make changes and resubmit.